.scrutinizer.yml

```diff
@@ -4,3 +4,4 @@ imports:
 tools:
     external_code_coverage:
         runs: 5 # No Code Coverage on PHP 5.2 and HHVM
+        timeout: 2700 # 45 minutes
```

Comment:
I had it at 25 mins earlier, but then realised that tests run for 20 minutes (wall time) and they might be queued for a bit too.